app/src/main/java/org/mozilla/focus/web/IWebViewLifecycleFragment.kt

```diff
@@ -77,13 +77,11 @@ abstract class IWebViewLifecycleFragment : LocaleAwareFragment() {
         super.onResume()
     }
 
-    override fun onDestroyView() {
-        super.onDestroyView()
-        if (webView != null) {
-            webView!!.callback = null
-            webView!!.destroy()
-            webView = null
-        }
+    override fun onDestroy() {
+        super.onDestroy()
```

Comment:
Hm, I think we shouldn't be constructing new WebView's in onCreateView if we're destroying them only in onDestroy – it seems like we'll have a memory leak. I filed a follow-up: https://github.com/mozilla-mobile/firefox-tv/issues/652